msi/release.sh

```diff
@@ -118,10 +118,35 @@ SCRIPT_PATH="$(cd "$(dirname "$0")" && pwd)"
 cd "$SCRIPT_PATH" ||
 die "Could not switch directory to $SCRIPT_PATH"
 
+# Generate the ReleaseNotes.html file
+test -f ../ReleaseNotes.html &&
+test ../ReleaseNotes.html -nt ../ReleaseNotes.md &&
+test ../ReleaseNotes.html -nt release.sh || {
+    # Install markdown
+    type markdown ||
+    pacman -Sy --noconfirm markdown ||
+    die "Could not install markdown"
+
+    (printf '%s\n%s\n%s\n%s %s\n%s %s\n%s\n%s\n%s\n' \
+        '<!DOCTYPE html>' \
+        '<html>' \
+        '<head>' \
+        '<meta http-equiv="Content-Type" content="text/html;' \
+        'charset=UTF-8">' \
+        '<link rel="stylesheet"' \
+        ' href="usr/share/git/ReleaseNotes.css">' \
+        '</head>' \
+        '<body class="details">' \
+        '<div class="content">'
+     markdown ../ReleaseNotes.md ||
+     die "Could not generate ReleaseNotes.html"
+     printf '</div>\n</body>\n</html>\n') >../ReleaseNotes.html
+}
+
```

Comment:
So, having an argument for the CSS location is "tricky" bordering on not worth it. The problem is, the script checks to make sure that:

1. `ReleaseNotes.html` doesn't already exist
2. `ReleaseNotes.html`, if it does exist, is not newer than `ReleaseNotes.md` or
3. `ReleaseNotes.html` is not newer than `<installer>/release.sh`.

If I were to add the option, then after generating the file the first time (using the proposed argument), subsequently creating the installer without making any changes to `ReleaseNotes.mds` or `<installer>/release.sh` would cause the 3 conditions above to all pass, and the file would _not_ be regenerated. Then, when we create the installer with the "stale" release notes file, it wouldn't be styled correctly because it would potentially be referencing an invalid CSS location.

OR, I could get rid of the checks above and _always_ regenerate the file. It's not like it takes that long....

It's your choice. If you want the option, the easiest thing would be to always regenerate the file. If you don't really want the option, I'll leave well enough alone.